src/ast/dibuilderbpf.cpp

```diff
@@ -39,12 +42,29 @@ void DIBuilderBPF::createFunctionDebugInfo(Function &func)
                                          DINode::FlagPrototyped,
                                          flags);
 
-  createParameterVariable(
-      subprog, "ctx", 1, file, 0, static_cast<DIType *>(types[1]), true);
+  for (size_t i = 0; i < args.fields.size(); i++) {
```

Comment:
nit: maybe this reads better:
`for (size_t i = 1; i < args.fields.size() + 1; i++) {`

Reply:
Ah yeah, you're right. Keep as is.